lab_statistics.Rmd

```diff
@@ -0,0 +1,287 @@
+---
+title: "Basic statistic"
+subtitle: "R Programming Foundation for Data Analysis"
+output:
+  bookdown::html_document2:
+    highlight: textmate
+    toc: true
+    toc_float:
+      collapsed: true
+      smooth_scroll: true
+      print: false
+    toc_depth: 4
+    number_sections: true
+    df_print: default
+    code_folding: none
+    self_contained: false
+    keep_md: false
+    encoding: 'UTF-8'
+    css: "assets/lab.css"
+    include:
+      after_body: assets/footer-lab.html
+---
+
+<br>
+
+```{r,child="assets/header-lab.Rmd"}
+```
+
+# Introduction
+
+In this lab, we will use statistics to gain a deeper understanding and insight into the dataset. Through statistical methods, we can evaluate data quality, identify patterns, and detect any outliers that may be present. You will apply several descriptive measures to explore and summarize the dataset, allowing for a more informed analysis.  
+
+# Generating the Data
+We create a dataset which consists of:  
+- Two numerical variables (Normally distributed).  
+- Three categorical. 
+
+**Note:** The following chunk has a function by which you generate a random dataset (`n` samples with `[0-100]%` **missing data**). You can also adjust these values to create a new dataset.  
```

Comment:
Maybe add here that we will go through functions the day later, so that they do not need to understand every line of it (yet ;) )

Reply:
Good point! I added.